playbooks/foreman.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
```

Comment:
Should we have this named `foreman_nightly` or just go with convention that unless a version is specified it is nightly? I'm OK with either, just want to clarify/establish.